docs/source/usage.rst

```diff
@@ -146,16 +150,32 @@ the ``--reveal-prefix`` command line flag to point to the local copy.
 
 This will create file ``your_talk.slides.html``, which you should be able to 
 access with ``open your_talk.slides.html``. To access the speaker notes, press 
-``s`` after the slides load and they should open in a new window. Keep in mind
-that if you want a functional timer inside the speaker notes, you need to serve
-the slides (see next paragraph for details).
+``s`` after the slides load and they should open in a new window.
+
+Note: a local copy of reveal.js does allow slides that will run completely
+offline, because the slides rely on other resources (e.g., mathjax and jquery)
+that still require access to public CDNs (by default).
+
+.. _ServePostProcessorExample:
+
+Serving slides with an https server: ``--post serve``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you have speaker notes working you may notice that your timers don't work.
+Timers require a bit more infrastructure; you need to serve your local copy of
+reveal.js from a local https server.
+
+Fortunately, ``nbconvert`` makes this fairly straightforward through the use of
+the ``ServePostProcessor``. To activate this server, we append the command line
+flag ``--post serve`` to our call to nbconvert. 
```

Comment:
Once you've got reveal.js set up next to the HTML file, I don't think you need the post processor - we could recommend `python3 -m http.server` to start a server.

If we can recommend this, it also gets us one step closer to getting rid of the postprocessor entirely.

Reply:
As you said, you don't *need* the post-processor, but it makes things easier for you.

I am +1 about the removal of post-processors, but I would keep the `--post serve` concept (maybe just calling it `serve` and implemented w/o post-processors) instead of recommending the `python3 -m http.server`.

Reply:
doesn't the simple http.server only serve http and not https? isn't it required to be an https server?

Reply:
Ok… well I just tested it and there is one tiny issue with `python3 -m http.server`, which is that it doesn't auto-open the file in the browser… meaning the person needs to navigate to `localhost:8000/your_talk.slides.html` in their browser which is both a context switch (cli to browser) and introduces a new layer of complexity (correctly arriving at the slides page relative to an implicit port statement). 

We can add the port declaration to the command `python3 -m http.server 8000` even if we keep it the default, but I still think that this is an additional opportunity for users to become confused.

Reply:
> meaning the person needs to navigate to localhost:8000/your_talk.slides.html in their browser which is both a context switch (cli to browser) and introduces a new layer of complexity (correctly arriving at the slides page relative to an implicit port statement).

I agree, this is why I would keep the `--post serve` until we provide a `--serve` option that is not based in a postprocessor.

Reply:
So… for this docs PR… I think we should still use `--post serve` @takluyver are you ok with that or do you feel strongly that we should include the slightly more cumbersome `python3 -m http.server` style instructions?